src/leaflet.js

```diff
@@ -160,12 +160,16 @@ export class LeafletCustomElement {
     };
     if (this.layers.hasOwnProperty('base')) {
       for (let layer of this.layers.base) {
-        layersToAttach.base[this.getLayerId(layer)] = this.layerFactory.getLayer(layer);
+        const id = this.getLayerId(layer);
+        if (this.attachedLayers.base.hasOwnProperty(id)) { continue; }
```

Comment:
could you change this to
```
if (!this.attachedLayers.base....)
```
So checking if it doesn't have this layer attached already and then add in instead of the `continue`?
Also: please put the block inside if to it's own line.

Same comment goes for the check for the overlay layers.